### PR TITLE
feat(core): start tracer with span attributes

### PR DESCRIPTION
**Pull request by hekike**
Default attributes for Tracer that will be added to every Span created by Tracer.
Useful for system level attributes like cluster information.

### CODE REVIEW

File: packages/opencensus-core/src/trace/config/types.ts [hunk 1]
```diff
@@ -32,6 +33,8 @@ export interface BufferConfig {
 
 /** Defines tracer configuration parameters */
 export interface TracerConfig {
+  /** A set of attributes each in the format [KEY]:[VALUE] */
+  attributes?: Attributes;
```
Comment (hekike) [inline packages/opencensus-core/src/trace/config/types.ts]: maybe calling it `spanAttributes`? `defaultSpanAttributes`?

File: packages/opencensus-core/src/trace/model/tracer-base.ts [hunk 2]
```diff
@@ -124,6 +124,12 @@ export class CoreTracerBase implements types.TracerBase {
       if (sampleDecision) {
         const rootSpan =
             new RootSpan(this, name, kind, traceId, parentSpanId, traceState);
+        // Add tracer attributes
+        if (this.config.attributes) {
+          for (const key of Object.keys(this.config.attributes)) {
```
Comment (mayurkale22) [inline packages/opencensus-core/src/trace/model/tracer-base.ts]: I would prefer to use `forEach` statement for better readability. WDYT?

File: packages/opencensus-core/src/trace/config/types.ts (hunk 1 above)
Comment (mayurkale22) [reply packages/opencensus-core/src/trace/config/types.ts]: Another option is either `constantAttributes` or `defaultAttributes`.

**Comment by hekike** [comment]
@mayurkale22 thanks for the review, I renamed `attributes` to `defaultAttributes` and refactored the code based on your comments.

File: packages/opencensus-core/src/trace/config/types.ts (hunk 1 above)
Comment (mayurkale22) [reply packages/opencensus-core/src/trace/config/types.ts]: Thanks for the change, would you mind to update CHANGELOG.md with this new feature?

**Comment by hekike** [comment]
Added to changelog.
*(For OpenTelemetry it would be useful to have automatic changelog generation via https://www.conventionalcommits.org)*

**Comment by mayurkale22** [comment]
> Added to changelog.
> _(For OpenTelemetry it would be useful to have automatic changelog generation via https://www.conventionalcommits.org)_

Not familiar with this tool, but happy to explore more. Thanks for sharing the link.

**Comment by hekike** [comment]
The links is just the spec for the commit message format.
The tools around it are:
https://github.com/conventional-changelog/conventional-changelog

For non-monorepo:
https://github.com/Netflix/unleash

etc.

**Pull request merged**